tests/lwip/ip.c

```diff
@@ -57,22 +57,26 @@ static void _ip_recv(sock_ip_t *sock, sock_async_flags_t flags, void *arg)
 
             printf("Received IP data from ");
             switch (src.family) {
-#if IS_USED(MODULE_LWIP_IPV4)
                 case AF_INET:
-                    printf("[%s]:\n",
-                           ipv4_addr_to_str(addrstr,
-                                            (ipv4_addr_t *)&src.addr.ipv4,
-                                            sizeof(addrstr)));
+                    if (IS_USED(MODULE_LWIP_IPV4)) {
+                        printf("[%s]:\n",
+                               ipv4_addr_to_str(addrstr,
+                                                (ipv4_addr_t *)&src.addr.ipv4,
+                                                sizeof(addrstr)));
+                        break;
+                    }
+                    printf("unspecified source\n");
```

Comment:
I used c-conditionals and it seems to work, should I change de message here to something like `inet unsupported`?

Reply:
since it continues the "Received IP data from" line, maybe say "unsupported protocol" ?